manifests/service.pp

```diff
@@ -0,0 +1,46 @@
+# Class: shibboleth_idp::service
+#
+# Manages the Jetty process
+#
+# Sample Usage:
+#
+#    sometype { 'foo':
+#      notify => Class['shibboleth_idp::service'],
+#    }
+#
+#
+class shibboleth_idp::service (
+  $service_name    = $::shibboleth_idp::params::service_name,
+  $service_enable  = $::shibboleth_idp::params::service_enable,
+  $service_ensure  = $::shibboleth_idp::params::service_ensure,
+  $service_manage  = $::shibboleth_idp::params::service_manage,
+  $service_restart = $::shibboleth_idp::params::service_restart,
+) {
```

Comment:
Why do we need params in this class? Do you expect anyone to use this class directly?
If you consider it private then just use the params from the init class and don't have params...